operations/security_and_authentication/Add_LDAP_User_Federation.md

```diff
@@ -48,9 +50,64 @@ LDAP user federation is not currently configured in Keycloak. For example, if it
    
 2. Update the LDAP settings in the customizations.yaml file.
 
-   1. (Optional) Add the LDAP CA certificate in the certs.jks section of customizations.yaml.
+   1. (Optional) Repopulate the keycloak_users_localize and cray-keycloak Sealed Secrets in the customizations.yaml file with the desired configuration.
+      
+      Check to see if the `generate:` sections of the Sealed Secrets have been populated with encrypted Sealed Secrets already:
+
+      ```bash
+      ncn-m001# yq write -i ./customizations.yaml 'spec.kubernetes.sealed_secrets.keycloak_users_localize.generate.data.(args.name==ldap_connection_url).args.value' "ldap://$LDAP"
```

Comment:
I don't think a `yq write` is going to check if the `generate:` sections of the Sealed Secrets have been populated with encrypted Sealed Secrets, but I'm no expert.

Also, this command is referring to a $LDAP environment variable that hasn't been set.